.github/workflows/main.yml

```diff
@@ -5,7 +5,7 @@ on:
   workflow_dispatch:
     inputs:
       updateImageTags:
-        description: "Placeholder to be used for deployment refactor"
+        description: "If true, the get_and_save_docker_image_tags job will run"
```

Comment:
As this is going to be read by potentially non-techie-team members, it's probably worth updating it to be something like: `If true, updates the Docker image tags for the webapp/service to the latest versions of the main branch` or something more succinct?